concepts/05 UI Components/TagBox/03 Overview/05 Elements.md

```diff
@@ -0,0 +1,9 @@
+<img src="/images/TagBox/tagbox-elements.png" alt="TagBox elements"/>
+
+1. [Label](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#label)
```

Comment:
we can add buttons that also can be shown, for example clear button and mark them as (optional) in the description